lua/windline/init.lua

```diff
@@ -198,6 +201,10 @@ end
 
 
 M.add_status = function(lines)
+    for _, line in ipairs(lines) do
+        line.inactive = line.in_active
+        line.show_while_active = line.show_in_active
+    end
```

Comment:
By just aliasing we avoid the confusion with the old names, performance wise is only while setting up and it's probably not measurable.

Reply:
This was wrong, if `line.inactive` was nil then it would break a status line using the new table keys.